dhall/tests/Dhall/Test/QuickCheck.hs

```diff
@@ -504,13 +542,24 @@ tests =
         , embedThenExtractIsIdentity (Proxy :: Proxy (Vector Double))
         , embedThenExtractIsIdentity (Proxy :: Proxy (Data.Map.Map Double Bool))
         , embedThenExtractIsIdentity (Proxy :: Proxy (HashMap.HashMap Double Bool))
+        , ( "Formatting should be idempotent"
+          , idempotenceTest
+          , Test.Tasty.adjustOption (const $ QuickCheckTests 1) -- TODO Increase this!
+          . adjustQuickCheckMaxSize 10000 -- This test discards many cases
```

Comment:
How does the max-size adjustment relate to the comment?

Reply:
Ah :sweat_smile: I meant to use `QuickCheckMaxRatio`! It's a bit of a weird name, but as far as I understand it corresponds to the number of allowed discarded tests.